Doc/library/asyncio-task.rst

```diff
@@ -843,6 +843,9 @@ Task Object
       .. versionchanged:: 3.9
          Added the *msg* parameter.
 
+      .. versionchanged:: 3.11
+         The ``msg`` parameter is propagated from cancelled task to its awaiter.
```

Comment:
Only the msg parameter or the exception? Is the latter an implementation detail?

Reply:
IIUC there are no guarantees about the identity of the exception object.

My personal opinion is that the whole "cancelled message" concept was a mistake, but it's too late to roll it back.

Reply:
I don't think `msg` needs to be mentioned in the `versionchanged`, IMO. The PR is more about eliminating unneeded intermediate `CancelledError`'s, resulting in a more compact traceback, etc. In other words, this PR would be useful even without the `msg` argument. It's more a side effect that (in most cases), the `msg` will bubble out.

Reply:
Could you propose better text, please?
The change is user-faced, the most visible difference is keeping/swallowing the cancellation message on crossing tasks border.

Reply:
Okay, then I'm okay with what you had.